data/KrishChopra22.json

```
@@ -0,0 +1,82 @@
+{
+  "name": "Krish Chopra",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source Enthusiast, Mobile App Developer",
+  "avatar": "https://user-images.githubusercontent.com/77012237/211071479-62ce9139-97e3-4314-844e-605575f69fb8.png",
```

It's best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/KrishChopra22.png",
```